index.html

```diff
@@ -72,6 +74,22 @@
 								that this has already happened.
 							</div>
 						</div>
+						<div class="tabs-item" data-tab="5">
+							<div class="tabs-item-title">Quote 5</div>
+							<div class="tabs-item-description">
+								There is a theory which states that if ever anyone discovers exactly what the Universe is for and why it is here, it will
+								instantly disappear and be replaced by something even more bizarre and inexplicable. There is another theory which states
+								that this has already happened.
+							</div>
+						</div>
+						<div class="tabs-item" data-tab="6">
+							<div class="tabs-item-title">Quote 6</div>
+							<div class="tabs-item-description">
+								There is a theory which states that if ever anyone discovers exactly what the Universe is for and why it is here, it will
+								instantly disappear and be replaced by something even more bizarre and inexplicable. There is another theory which states
+								that this has already happened.
+							</div>
+						</div>
 					</div>
 				</div>
```

Comment:
- Great job on the assingment Haja. You completed the MVP by implementing Dropdown functionality along with Tabs components.
- Your code is clean and git workflow is fluid as usual, great job 👍